builder/amazon/common/run_config_test.go

```diff
@@ -52,6 +59,25 @@ func TestRunConfigPrepare_SourceAmi(t *testing.T) {
 	}
 }
 
+func TestRunConfigPrepare_SourceAmiFilterBlank(t *testing.T) {
+	c := testConfigFilter()
+	if err := c.Prepare(nil); len(err) != 1 {
+		t.Fatalf("err: %s", err)
+	}
+}
+
+func TestRunConfigPrepare_SourceAmiFilterGood(t *testing.T) {
+	c := testConfigFilter()
+	owner := "123"
+	filter_key := "name"
+	filter_value := "foo"
```

Comment:
this is totally fine, but fyi there's a helper method in aws [`aws.String()`](https://github.com/aws/aws-sdk-go/blob/master/aws/convert_types.go#L6)